python/ray/data/tests/test_map.py

```diff
@@ -1057,6 +1058,39 @@ def test_nonserializable_map_batches(shutdown_only):
         x.map_batches(lambda _: lock).take(1)
 
 
+def test_map_batches_async_generator(shutdown_only):
+    ray.shutdown()
+    ray.init(num_cpus=10)
+
+    async def sleep_and_yield(i):
+        print("sleep", i)
+        await asyncio.sleep(i % 5)
+        print("yield", i)
+        return {"input": [i], "output": [2**i]}
+
+    class AsyncActor:
+        def __init__(self):
+            pass
+
+        async def __call__(self, batch):
+            tasks = [sleep_and_yield(i) for i in batch["id"]]
```

Comment:
this line is redundant.

Reply:
thanks for the catch, i had thought i removed it but somehow ended up back again...